src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -92,70 +143,37 @@ def test_WINNF_FT_S_FAD_1(self):
-                    self.assertEqual(record['grants'][0]['id'],grant_response[index]['grantId'])
-                    if record['grants'][0]['operationParam'] != null :
-                        self.assertEqual(record['grants'][0]['operationParam']['maxEirp'],\
-                                         grants_to_request[index]['operationParam']['maxEirp'])    
-                        self.assertEqual(record['grants'][0]['operationParam'], ['operationFrequencyRange']['lowFrequency'],\
-                                         grants_to_request[index]['operationParam']['operationFrequencyRange']['lowFrequency'])
-                        self.assertEqual(record['grants'][0]['operationParam']['operationFrequencyRange']['highFrequency'],\
-                                         grants_to_request[index]['operationParam']['operationFrequencyRange']['highFrequency'])
-                    self.assertEqual(record['grants'][0]['requestedOperationParam']['maxEirp'],\
-                                     grants_to_request[index]['requestedOperationParam']['maxEirp'])    
-                    self.assertEqual(record['grants'][0]['requestedOperationParam']['operationFrequencyRange']\
-                                     ['lowFrequency'],grants_to_request[index]['requestedOperationParam']['operationFrequencyRange']['lowFrequency'])
-                    self.assertEqual(record['grants'][0]['requestedOperationParam']\
-                                     ['operationFrequencyRange']['highFrequency'], grants_to_request[index]['requestedOperationParam']\
-                                     ['operationFrequencyRange']['highFrequency'])
-                    self.assertEqual(record['grants'][0]['channelType'], grant_response[index]['channelType'])
-                    self.assertEqual(record['grants'][0]['grantExpireTime'], grant_response[index]['grantExpireTime'])
-                    self.assertEqual(False, record['grants'][0]['terminated']) 
-            elif activity_dump_file['recordType'] == 'esc_sensor':
-                self.assertEqual(1, len(data['recordData']))
-                # Verify the response file of Esc Sensor
-                self.assertContainsRequiredFields("EscSensorRecord.schema.json", data['recordData'][0])
-                self.assertDictEqual(esc_sensor_record, data['recordData'][0])
-            else:
-                self.assertEqual(activity_dump_file['recordType'],'zone')
-                self.assertEqual(1, len(data['recordData']))
-                # Verify the response file of PPA
-                self.assertContainsRequiredFields("zoneData.schema.json", data['recordData'][0])
-                self.assertDictEqual(ppa_record, data['recordData'][0] )
-                 
-                                                  
\ No newline at end of file
+        self.assertEqual(3, len(response['files']))
+        cbsd_dump_file = [file for file in response['files'] if file['recordType'] ==  'cbsd']
```

Comment:
The name `cbsd_dump_files` (plural) is probably more accurate since there may be more than one.